LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2020-present Burkhard Reffeling, Maciej Malkowski, McKayla Washburn
```

Comment:
Should we change this to "contributors" or something more generic? Or are we gonna update this everytime there's a new contributor? If so, then Roman and myself are omitted :(

Reply:
@victorhqc I think copyrights cannot be owned by something generic like contributors - maybe we should use Klarna legal entity as the copyright holder?

Reply:
Why not as "Github contributors" or "Repository contributors".

Reply:
https://github.com/klarna/electron-redux/graphs/contributors

Reply:
Cause contributors have to be defined by name right? We cannot have implicit list in a license. Usually you would have a CLA signed before contributing so we don't append every single person to that list. Maybe lets add the ones that we think should be there, like you mentioned you and Roman? @victorhqc